client/my-sites/themes/theme-showcase-header.jsx

```diff
@@ -53,16 +54,31 @@ export default function ThemeShowcaseHeader( { canonicalUrl, filter, tier, verti
 	return (
 		<>
 			<DocumentHead title={ documentHeadTitle } meta={ metas } />
-			<ThemesHeader title={ themesHeaderTitle } description={ themesHeaderDescription }>
-				{ isLoggedIn && (
-					<>
-						<div className="themes__install-theme-button-container">
-							<InstallThemeButton />
-						</div>
-						<ScreenOptionsTab wpAdminPath="themes.php" />
-					</>
-				) }
-			</ThemesHeader>
+			{ isLoggedIn ? (
+				<NavigationHeader
+					compactBreadcrumb={ false }
+					navigationItems={ [] }
+					mobileItem={ null }
+					title={ translate( 'Themes' ) }
+					subtitle={ translate(
+						'Select or update the visual design for your site. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
+						{
+							components: {
+								learnMoreLink: <InlineSupportLink supportContext="themes" showIcon={ false } />,
+							},
+						}
+					) }
+				>
```

Comment:
Does `NavigationHeader` maintain the `ScreenOptionsTab` internally?

Reply:
Good catch, apparently not, I will check it right now!

Reply:
I created this follow-up: https://github.com/Automattic/dotcom-forge/issues/4229